src/main/java/com/teamscale/buildbreaker/commandline/FindingEvalOptions.java

```diff
@@ -0,0 +1,55 @@
+package com.teamscale.buildbreaker.commandline;
+
+import picocli.CommandLine;
+
+import static com.teamscale.buildbreaker.commandline.BranchAndTimestampUtils.validateBranchAndTimestamp;
+
+class FindingEvalOptions {
```

Comment:
Extracted from BuildBreaker. Also added 4 new options:
* --target-revision
* --target-branch-and-timestamp
* --base-revision
* --base-branch-and-timestamp